### ISSUE TITLE
Code release

### ISSUE THREAD

**Issue by Senwang98** [njulj/RFANet#5]
Hi, @njulj 
Thanks for sharing this wonderful work. I have some questions:
1. Which validation set have you used in training processing, I mean Set5 or DIV2K 801-810 or Set14.
2. When will you update the code in github, I am look forward to your code. You build your model based on EDSR-Pytorch?
